src/domains/fdr-app/api/fdr-fase1/nodoPerPsp/v1/fdr_nodoinvia_flussorendicontazione_flow.xml

```diff
@@ -80,7 +80,47 @@
                       </send-one-way-request>
                   </when>
                   <otherwise>
-                      <set-backend-service base-url="${base-url}" />
+                      <set-variable name="enable_fdr_psp_soap_request_switch" value="{{enable-fdr-psp-soap-request-switch}}" />
+                      <choose>
+                        <when condition="@(context.Variables.GetValueOrDefault<string>("enable_fdr_psp_soap_request_switch", "").Equals("true") )">
```

Comment:
Ditto

Reply:
why do we need https://pagopa.atlassian.net/wiki/spaces/IQCGJ/pages/1071153182/FdR-1+Flussi+di+Rendicontazione#API---Nodo-Invia-Flusso-Rendicontazione-FTP to forward request to nexi ?